bundles/org.openhab.binding.wundergroundupdatereceiver/src/main/java/org/openhab/binding/wundergroundupdatereceiver/internal/WundergroundUpdateReceiverServlet.java

```diff
@@ -0,0 +1,196 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.wundergroundupdatereceiver.internal;
+
+import static java.util.stream.Collectors.toMap;
+import static java.util.stream.Collectors.toSet;
+import static org.openhab.binding.wundergroundupdatereceiver.internal.WundergroundUpdateReceiverBindingConstants.*;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.Dictionary;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Hashtable;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.core.io.http.servlet.BaseOpenHABServlet;
+import org.osgi.service.http.HttpContext;
+import org.osgi.service.http.HttpService;
+import org.osgi.service.http.NamespaceException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link WundergroundUpdateReceiverServlet} is responsible for receiving updates,and
+ * updating the matching channels.
+ *
+ * @author Daniel Demus - Initial contribution
+ */
+@NonNullByDefault
+public class WundergroundUpdateReceiverServlet extends BaseOpenHABServlet {
+
+    public static final String SERVLET_URL = "/weatherstation/updateweatherstation.php";
+    private static final long serialVersionUID = -5296703727081438023L;
```

Comment:
```suggestion
    private static final long SERIAL_VERSION_UID = -5296703727081438023L;
```
See https://www.openhab.org/docs/developer/guidelines.html#java-coding-style
Constants should be in this format so they can be easily recognized.

Reply:
You do realize that this is a requirement of java serialization and implementing Serializable since at least j2se 1.2 in 1998? It masks the serialVersionUID inherited from BaseOpenHABServlet and would probably crash openhab every time it tries to deserialize the servlet (if it serializes it at all). So no I don't think it needs to follow other guidelines than the java object serialization specification.